src/apps/front-office/design-system/layouts/Footer/Footer.tsx

```diff
@@ -1,3 +1,145 @@
+import { Link } from "@mongez/react-router";
+import appleIcon from "assets/images/apple-icon-store.jpg";
```

Comment:
I'd recommend creating/finding an `assets` file that exports all assets images from there.